### DISCUSSION TITLE
Suppress tree-shaking warning from optimizer-css?

### DISCUSSION THREAD

**Discussion by dannebas** [parcel-bundler/parcel#8942] [Q&A] [answered]
I am using scss modules in a React application together with jest and storybook and need to be able to use default import syntax for my .module.scss files.
Is there a way to configure Parcel to ignore tree-shaking for those files? --no-scope-hoist seems to work but I assume that will disable it for all files?
I have tried adding sideEffects: ["*.scss"] to package.json without success..

**Comment by mischnic** [comment, marked as answer]
No, there unfortunately isn't. Parcel is missing some warning-suppression functionality in general (also for transformers, e.g. for [suboptimal tree shaking](https://parceljs.org/features/scope-hoisting/#avoiding-bail-outs))

**Reply by dannebas** [reply]
Thank you. I guess I will have to live with the warnings for now. :)
